rma_purchase/models/rma_order_line.py

```diff
@@ -0,0 +1,253 @@
+# Copyright 2017-2022 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class RmaOrderLine(models.Model):
+    _inherit = "rma.order.line"
+
+    def _compute_purchase_count(self):
+        for rec in self:
+            purchase_line_count = self.env["purchase.order.line"].search(
+                [("rma_line_id", "=", rec.id)]
+            )
+            rec.purchase_count = len(purchase_line_count.mapped("order_id"))
+
+    def _compute_purchase_order_lines(self):
+        for rec in self:
+            purchase_list = []
+            for line in self.env["purchase.order.line"].search(
+                [("rma_line_id", "=", rec.id)]
+            ):
+                purchase_list.append(line.id)
+            rec.purchase_order_line_ids = [(6, 0, purchase_list)]
+
+    def _compute_qty_purchase(self):
```

Comment:
```suggestion

    @api.depends("operation_id","purchase_policy")
    def _compute_qty_purchase(self):
```
Perhaps it may be necessary to include additional fields within the 'depends' section.

Reply:
Done!